docs/conf.py

```diff
@@ -345,10 +345,13 @@
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
-    "python": ("http://python.readthedocs.org/en/latest/", None),
-    "google-auth": ("https://google-auth.readthedocs.io/en/stable", None),
+    "python": ("https://python.readthedocs.org/en/latest/", None),
+    "google-auth": (
+        "https://googleapis.dev/python/google-auth/latest/index.html",
```

Comment:
[Breaks the `docs` build](https://source.cloud.google.com/results/invocations/df5b7042-befa-4a14-bf50-bd972cede9a5/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-firestore%2Fpresubmit%2Fpresubmit/log).This is supposed to be the directory from which `objects.inv` can be fetched.
```suggestion
        "https://googleapis.dev/python/google-auth/latest/",
```

Reply:
Fixed in https://github.com/googleapis/synthtool/pull/860/.

Reply:
I checked the "regenerate this PR" box.  Let's see how that goes.